index.html

```diff
@@ -10,7 +10,9 @@
     <meta name="author" content="">
 
     <title>cardinal designs</title>
-
+    <META NAME="description" CONTENT="Cardinal Designs">
+    <META NAME="keyword" CONTENT="Graphic Design, South Carolina, CHarleston, Park Circle, Advertising. Web Design, Branding, Logos">
```

Comment:
I caught that as well. I’m going to fix that.

Jason Bremer
843.670.8996
cardinaldesigns.info





> On Aug 24, 2015, at 12:55 PM, joshuabremer <notifications@github.com> wrote:
> 
> In index.html <https://github.com/jcbremer/cardinaldesign/pull/21#discussion_r37773604>:
> 
> > @@ -10,7 +10,9 @@
> >      <meta name="author" content="">
> >  
> >      <title>cardinal designs</title>
> > -
> > +    <META NAME="description" CONTENT="Cardinal Designs">
> > +    <META NAME="keyword" CONTENT="Graphic Design, South Carolina, CHarleston, Park Circle, Advertising. Web Design, Branding, Logos">
> Try not to use uppercase elements. It's better style to use lower case:
> 
> <meta name="keyword" content="....">
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/jcbremer/cardinaldesign/pull/21/files#r37773604>.
>